Frameworks/include/CGImageInternal.h

```diff
@@ -34,93 +40,209 @@ class DisplayTexture;
 
 class DisplayTextureLocking {
 public:
-    virtual void* LockWritableBitmapTexture(DisplayTexture* tex, int* stride) = 0;
-    virtual void UnlockWritableBitmapTexture(DisplayTexture* tex) = 0;
-    virtual void RetainDisplayTexture(DisplayTexture* tex) = 0;
-    virtual void ReleaseDisplayTexture(DisplayTexture* tex) = 0;
+  virtual void *LockWritableBitmapTexture(DisplayTexture *tex, int *stride) = 0;
```

Comment:
seems so.

---
In reply to: [84145705](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84145705) [](ancestors = 84145705)